save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/internal/UsersController.kt

```diff
@@ -34,40 +34,46 @@ class UsersController(
     private val objectMapper = ObjectMapper()
         .findAndRegisterModules()
         .registerModule(CoreJackson2Module())
-        .addMixIn(IdentitySourceAwareUserDetails::class.java, IdentitySourceAwareUserDetailsMixin::class.java)
+        .addMixIn(IdAwareUserDetails::class.java, IdentitySourceAwareUserDetailsMixin::class.java)
 
     /**
-     * Stores [user] in the DB
+     * Stores user in the DB with provided [name] with [authorities] as role.
+     * And add a link to [source] for created user
      *
-     * @param user user to store
+     * @param source user source
+     * @param name user name
+     * @param authorities
      */
-    @PostMapping("/new")
+    @PostMapping("/{source}/{name}")
```

Comment:
I think its better to leave `/new` in endpoint

Reply:
returned back